tests/js/common/aql/aql-search-optimize-top-k.js

```diff
@@ -65,42 +71,213 @@ function testOptimizeTopK() {
 
       // testWithoutScore
       for (const v of names) {
-        const query = "FOR d IN " + v + " SEARCH d.f == 'a' OPTIONS {waitForSync:true} LIMIT 10 RETURN d";
-        print(query);
+        const query = "FOR d IN " + v + " SEARCH d.f == 'a' OPTIONS {waitForSync:true} LIMIT 100 RETURN d";
         let res = db._query(query).toArray();
-        assertEqual(res.length, 10);
+        assertEqual(res.length, 100);
+      }
+
+      for (const v of names) {
```

Comment:
What are we testing here? One round without WAND to check sanity is ok. But what is behind repeating queries with different LIMIT?

Reply:
You have asked about something like this during previous review:
```
Here we need to test potential issues with AQL batch switching. And it is needed in PR runs. Benchmarks are not the question here. Is it possible to just auto-generate 2000-3000 documents and run query that will return around 1500? If do batch-inserts it should be quite fast.
```

Reply:
Ah, forgot to add scorers. Will fix